README.md

```diff
@@ -1,12 +1,19 @@
 <p style="text-align:center;" align="center">
-  <img align="center" src="https://raw.githubusercontent.com/layer5io/layer5/master/assets/images/layer5/layer5-tag-community-white-bg.png" width="45%" /></p>
-  
-![GitHub contributors](https://img.shields.io/github/contributors/layer5io/layer5.svg)
-![GitHub](https://img.shields.io/github/license/layer5io/layer5.svg) 
-![GitHub issues by-label](https://img.shields.io/github/issues/layer5io/layer5/help%20wanted.svg?color=%23DDDD00)
-[![Slack](http://slack.layer5.io/badge.svg)](http://slack.layer5.io)
-![Twitter Follow](https://img.shields.io/twitter/follow/layer5.svg?label=Follow&style=social)
-  
+  <img align="center" src="https://raw.githubusercontent.com/layer5io/layer5/master/src/assets/images/layer5/layer5-tagline/png/layer5-tag-community-white-bg.png" width="45%" /></p>
+
+<p align="center">
+<a href="https://github.com/layer5io/layer5" alt="GitHub contributors">
+<img src="https://img.shields.io/github/contributors/layer5io/layer5.svg" /></a>
+<a href="https://github.com/layer5io/layer5" alt="License">
+<img src="https://img.shields.io/github/license/layer5io/layer5.svg" /></a>
+<a href="https://github.com/layer5io/layer5" alt="GitHub issues by-label">
+<img src="https://img.shields.io/github/issues/layer5io/layer5/help%20wanted.svg?color=%23DDDD00" /></a>
+<a href="https://github.com/layer5io/layer5" alt="Slack">
+<img src="https://img.shields.io/badge/Slack-@layer5.svg?logo=slack" /></a>
```

Comment:
@singh08prashant : Can you pl rework on it? [Only change the slack badge i.e, failing]
For place it as `center`, Pl take the reference from <a href="https://github.com/service-mesh-performance/service-mesh-performance/pull/196/commits/4ce829327186b4b9a56002fd49a554a5266437ed">here</a>.
Else Good! :slightly_smiling_face:

Reply:
Thank you! I'll make changes accordingly.